includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -2099,6 +2102,9 @@ public static function render_single_url_error_details( $validation_error, $term
 				</p>
 			</dd>
 
+			<dt><?php esc_html_e( 'Error code', 'amp' ); ?></dt>
+			<dd><code><?php echo esc_html( $validation_error['code'] ); ?></code></dd>
```

Comment:
> My favorite would be `<?= esc_html() ?>`,

Don't you mean `<?= esc_html__() ?>`?

Reply:
Actually, the latter is `esc_html()` because it is not using a translated string. I think I misunderstood your consistency request. So eventually, you'd like:

`<?= esc_html__() ?>` with `<?= esc_html() ?>`

rather than

`<?php esc_html_e() ?>` with `<?php echo esc_html() ?>`